pkgs/development/web/postman/default.nix

```
@@ -1,18 +1,53 @@
-{ stdenv, lib, gnome2, fetchurl, pkgs, xorg, makeWrapper, makeDesktopItem }:
+{ stdenv, lib, fetchurl, pkgs, makeWrapper, makeDesktopItem, wrapGAppsHook
+, atk, at-spi2-atk, alsaLib, cairo, cups, dbus, expat, gdk-pixbuf, glib, gtk3
+, freetype, fontconfig, nss, nspr, pango, udev, libX11, libxcb, libXi
+, libXcursor, libXdamage, libXrandr, libXcomposite, libXext, libXfixes
+, libXrender, libXtst, libXScrnSaver
+}:
+
+with stdenv.lib;
```

file wide "with's" should be discouraged. Since you're also receiving lib, you can just use "lib"
```suggestion
with stdenv.lib;
```